tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -209,7 +209,7 @@ $ ls ./proto/myChain | xargs -I {} protoc \
 $ ls ./proto/myChain | xargs -I {} \
     docker run --rm -i \
     -v $(pwd):/project -w /project \
-    ts-protoc \
+    ts-proto \
```

Comment:
```suggestion
    ts-protoc \
```

It is the name of the Docker image. It was created on line 102 and is also used for instance on line 127.

Reply:
Ah, okay. You said "yes" in your explanation, which I thought meant "yes to your correction"...

Reply:
Oopsie I read it as "is this correct?".